src/codegen/llvm/llvm_module.cc

```diff
@@ -62,6 +63,11 @@ class LLVMModuleNode final : public runtime::ModuleNode {
       return PackedFunc([flag](TVMArgs args, TVMRetValue *rv) {
           * rv = flag;
         });
+    } else if (name == "get_target_triple") {
```

Comment:
consider rename to _get_target_triple just to avoid potential user defined function.

Reply:
Will do